types/src/v0/v0_1/chain_config.rs

```diff
@@ -40,6 +40,9 @@ pub struct ChainConfig {
     /// regardless of whether or not their is a `fee_contract` deployed. Once deployed, the fee
     /// contract can decide what to do with tokens locked in this account in Espresso.
     pub fee_recipient: FeeAccount,
+
+    /// Account that receives sequencing bids.
+    pub bid_recipient: FeeAccount,
```

Comment:
This is a breaking change so shouldn't we create a new v3 chain config and use that ?

Reply:
originally fixed in fd74b19081f592d7b639d548e8e6be72320001d5 and kinks mostly worked out in f1be76370075bf46a89946db4d7c725745bfb6ad